components/dashboard/Sidebar/index.tsx

```diff
@@ -37,7 +37,8 @@ const Sidebar = ({
   const [mounted, setMounted] = useState(false);
   const [createPostLoading, setCreatePostLoading] = useState(false);
 
-  const [postPublic, setPostPublic] = useAtom(postPublicAtom);
+  const [postPublic, setPostPublic] = useAtom(selectedNoteType);
```

Comment:
This is incorrect. It should be called `selectedNote`, not `postPublic`. The value is `selectedNote.isPublic`.